### ISSUE TITLE
Revision request for splitmix

### ISSUE THREAD

**Issue by athas** [haskell-infra/hackage-trustees#340]
Needs to have its upper bound on `base` bumped.

[Upstream PR](https://hackage.haskell.org/package/splitmix)

**Comment by sjakobi** [comment]
I assume you meant to link to https://github.com/haskellari/splitmix/pull/65.

I think it's a bit early to request trustee action here – GHC 9.4.1 is very fresh and the `splitmix` maintainer didn't have even half an hour to respond to your PR yet.

I'll tentatively close this issue for now. Feel free to reopen this issue once the maintainer hasn't taken action for two weeks or so.

**Comment by athas** [comment]
![image](https://user-images.githubusercontent.com/55833/183467305-9127d97f-35bc-4b3b-a6a4-28257ed42f88.png)

**Comment by phadej** [comment]
I must admit seeing this ate a bit of my motivation and excitement about GHC-9.4.

**Comment by andreasabel** [comment]
> I must admit seeing this ate a bit of my motivation and excitement about GHC-9.4.

@phadej, what do you mean?  
Folks are eager to jump on the train, and quite a few packages at the root of the dependency tree are in your care...

**Comment by phadej** [comment]
> Folks are eager to jump on the train

I'm not. The release came out (again) as somewhat a surprise. I have other plans for this week (i.e. normal work) and weekend (something else).

If people don't care to read through `CONTRIBUTING.md` and open trustee tickets immediately, I don't know why I would spoil them any more by reacting to the GHC release ASAP.

**Comment by phadej** [comment]
FYI, seems that haskell/action/setup doesn't use ghcup on windows, so I'm somewhat blocked on that.

AFAIK Windows require yet to be made `cabal-install` release as GHC stuff on Windows has changed significantly, that is another issue which might be a blocker.

I'd be happier to use `cabal-install-3.8` on other platforms as well, as

```
Warning: Unknown/unsupported 'ghc' version detected (Cabal 3.6.2.0 supports
'ghc' version < 9.4): /opt/ghc/bin/ghc-9.4.1 is version 9.4.1
```

is not nice to see.

**Comment by andreasabel** [comment]
Thanks for the swift revision of `splitmix` (and many other crucial packages), @phadej!
